src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -232,20 +235,32 @@ public static Date parseDate(String date) throws ParseException {
      * Parses a {@code String description} into a {@code Description}.
      * Leading and trailing whitespaces will be trimmed.
      */
-    public static IterationDescription parseIterationDescription(String description) throws ParseException {
+    public static IterationDescription parseIterationDescription(String description) {
         requireNonNull(description);
         String trimmedDescription = description.trim();
         return new IterationDescription(trimmedDescription);
     }
 
     /**
-     * Parses a {@code String description} into a {@code Description}.
+     * Parses a {@code String imagePath} into a {@code ImagePath}.
      * Leading and trailing whitespaces will be trimmed.
      */
     public static ImagePath parseImagePath(String imagePath) throws ParseException {
         requireNonNull(imagePath);
         String trimmedImagePath = imagePath.trim();
-        return new ImagePath(trimmedImagePath);
+        Path srcPath = Path.of(trimmedImagePath);
+        if (Files.exists(srcPath)) {
+            try {
+                System.out.println(Files.probeContentType(srcPath));
```

Comment:
Hm, just curious as to why this print statement is here- is this intentional?

Reply:
Oh lol forgot to remove 😓